tests/test_consolidated_commands.py

```
@@ -0,0 +1,503 @@
+
+def test_browse_branches_integration(monkeypatch, tmp_path, capsys):
+    """Test browse branches command end-to-end."""
+    _configure_isolated_state(monkeypatch, tmp_path)
+    monkeypatch.setenv("POLYLOGUE_FORCE_PLAIN", "1")
+    monkeypatch.setattr(sys, "argv", ["polylogue", "browse", "branches", "--no-picker"])
+
+    main()
+
+    # Should not crash
+    output = capsys.readouterr()
+    assert output or True  # Just verify it runs
```

Ruff (0.15.9)

[warning] Use `True` instead of `... or True`

Replace with `True`

(SIM222)